chirp/drivers/anytone.py

```diff
@@ -733,14 +753,3 @@ class PolmarDB50MRadio(AnyTone5888UVRadio):
     VENDOR = "Polmar"
     MODEL = "DB-50M"
     _file_ident = ["DB-50M"]
-
-
-@directory.register
-class PowerwerxDB750XRadio(AnyTone5888UVRadio):
-    """Powerwerx DB-750X"""
-    VENDOR = "Powerwerx"
-    MODEL = "DB-750X"
-    _file_ident = ["DB-750X"]
-
-    def get_settings(self):
-        return {}
```

Comment:
Why are you removing this? It will break anyone that uses this subclass and has files with that model in the metadata.